sdk/metricsadvisor/ai-metrics-advisor/review/ai-metrics-advisor.api.md

```diff
@@ -0,0 +1,1152 @@
+    deleteDataFeed(dataFeedId: string, options?: OperationOptions): Promise<RestResponse>;
+    deleteHook(hookId: string, options?: OperationOptions): Promise<RestResponse>;
+    deleteMetricAnomalyDetectionConfiguration(detectionConfigurationId: string, options?: OperationOptions): Promise<RestResponse>;
+    readonly endpointUrl: string;
+    getAnomalyAlertConfiguration(alertConfigurationId: string, options?: OperationOptions): Promise<GetAnomalyAlertConfigurationResponse>;
+    getDataFeed(dataFeedId: string, options?: OperationOptions): Promise<GetDataFeedResponse>;
+    getDataFeedIngestionProgress(dataFeedId: string, options?: {}): Promise<GeneratedClientGetIngestionProgressResponse>;
+    getHook(hookId: string, options?: OperationOptions): Promise<GetHookResponse>;
+    getMetricAnomalyDetectionConfiguration(detectionConfigurationId: string, options?: OperationOptions): Promise<GetAnomalyDetectionConfigurationResponse>;
+    listAnomalyAlertConfigurations(detectionConfigurationId: string, options?: OperationOptions): PagedAsyncIterableIterator<AnomalyAlertConfiguration, ListAnomalyAlertConfigurationsPageResponse, undefined>;
+    listDataFeedIngestionStatus(dataFeedId: string, startTime: Date, endTime: Date, options?: ListDataFeedIngestionStatusOptions): PagedAsyncIterableIterator<IngestionStatus, ListDataFeedIngestionStatusPageResponse>;
+    listDataFeeds(options?: ListDataFeedsOptions): PagedAsyncIterableIterator<DataFeed, ListDataFeedsPageResponse>;
+    listHooks(options?: ListHooksOptions): PagedAsyncIterableIterator<HookUnion, ListHooksPageResponse>;
+    listMetricAnomalyDetectionConfigurations(metricId: string, options?: OperationOptions): PagedAsyncIterableIterator<AnomalyDetectionConfiguration, ListAnomalyDetectionConfigurationsPageResponse, undefined>;
+    refreshDataFeedIngestion(dataFeedId: string, startTime: Date, endTime: Date, options?: OperationOptions): Promise<RestResponse>;
+    updateAnomalyAlertConfiguration(alertConfigurationId: string, patch: AnomalyAlertConfigurationPatch, options?: OperationOptions): Promise<GetAnomalyAlertConfigurationResponse>;
+    updateDataFeed(dataFeedId: string, patch: DataFeedPatch, options?: OperationOptions): Promise<GetDataFeedResponse>;
+    updateHook(hookId: string, patch: EmailHookPatch | WebhookHookPatch, options?: OperationOptions): Promise<GetHookResponse>;
+    updateMetricAnomalyDetectionConfiguration(detectionConfigurationId: string, patch: AnomalyDetectionConfigurationPatch, options?: OperationOptions): Promise<GetAnomalyDetectionConfigurationResponse>;
+}
+
+// @public
+export interface MetricsAdvisorAdministrationClientOptions extends PipelineOptions {
+}
+
+// @public
+export class MetricsAdvisorClient {
+    constructor(endpointUrl: string, credential: MetricsAdvisorKeyCredential, options?: MetricsAdvisorClientOptions);
+    createMetricFeedback(feedback: MetricFeedbackUnion, options?: {}): Promise<GetFeedbackResponse>;
```

Comment:
shouldn't the options bag have a type? I assume it'd be at least OperationOptions

Reply:
Yes it should.